Makefile

```diff
@@ -1,5 +1,5 @@
-PREFIX?=wavefronthq
-DOCKER_IMAGE?=wavefront-kubernetes-collector
+PREFIX?=projects.registry.vmware.com/tanzu_observability_keights_saas
```

Comment:
I think this url can be assumed by CI if image has "*-snapshot" as its suffix , and avoid exposing this registry path.

Reply:
The goal was to use the same make / scripts locally to test and build as we do on CI.

 **projects.registry.vmware.com/tanzu_observability_keights_saas registry** is our dev public registry. Do you think it's a concern highlighting it in the make file?